### ISSUE TITLE
We accidentally wrote over the token login method

### ISSUE THREAD

**Issue by 808sAndBR** [tcash21/zendeskR#7]
During the roxygen rewrite, I didn't think about sequencing the PRs well and accidentally overwrote the log in with the token method. I now have the tokenless version in a few branches I'm working but will open another PR to reimplement it once everything else is merged.

**Comment by Connor1462** [comment]
Hey! Did you ever fix token method login? I'm just discovering this package and I'd love to use it!
